<ui version="4.0" >
 <class>EntryDialog</class>
 <widget class="QDialog" name="EntryDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>171</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Legend:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="legend" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="type" />
     </item>
     <item row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Start date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QDateTimeEdit" name="startDate" >
       <property name="calendarPopup" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Completion:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="completion" >
       <property name="suffix" >
        <string>%</string>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>End date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" >
      <widget class="QDateTimeEdit" name="endDate" >
       <property name="calendarPopup" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QCheckBox" name="readOnly" >
       <property name="text" >
        <string>Read Only</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QLabel" name="dependsLabel" >
       <property name="text" >
        <string>Depends on:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" >
      <widget class="QComboBox" name="depends" >
       <item>
        <property name="text" >
         <string>- None -</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
